s12/protobuf/proto/validator.proto

```diff
@@ -32,4 +32,8 @@ extend google.protobuf.FieldOptions {
   optional bool legacy_id = 65210;
   // Supplementary validation option to trim leading and trailing whitespaces (as defined by Unicode) before doing length check
   optional bool trim_len_check = 65211;
+  // collection size lesser than or equal this value
+  optional int64 collection_size_gte = 65212;
```

Comment:
I'd rather keep Go or Protobuf terminology for consistency. suggestion: `array_len_gte`, `slice_len_gte` or `repeated_len_gte`

Reply:
good idea. going for `repeated_len_gte` to be consistent with proto terms.